builtin/settingtypes.txt

```diff
@@ -1319,6 +1319,12 @@ language (Language) enum   ,be,ca,cs,da,de,dv,en,eo,es,et,fr,he,hu,id,it,ja,jbo,
 #    -    verbose
 debug_log_level (Debug log level) enum action ,none,error,warning,action,info,verbose
 
+#    If the file size of debug.txt exceeds the number of bytes specified in
```

Comment:
Such large numbers are not user-friendly. The size in megabytes would be easier to understand for normal users. `50E6` would be possible here, but you cannot expect this notation from regular users.

Reply:
I've changed it to megabytes.